docker-compose.override.dev.yml

```diff
@@ -8,7 +8,12 @@ services:
     environment:
       DD_DEBUG: 'True'
       DD_ADMIN_USER: ${DD_ADMIN_USER:-admin}
-      DD_ADMIN_PASSWORD: ${DD_ADMIN_PASSWORD:-admin}      
+      DD_ADMIN_PASSWORD: ${DD_ADMIN_PASSWORD:-admin}
+    ports:
+      - target: 3000
+        published: 3000
```

Comment:
Could we eventually make this `${DD_DEBUG_PORT:-3000}`

Reply:
Changed, thanks :+1: